CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Change Log
 
+## [1.0.0rc1](https://github.com/stellargraph/stellargraph/tree/v1.0.0rc1)
+
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.11.0...v1.0.0rc1)
+
+### Major features and improvements
+
+- Efforts to improve the getting-started process have continued, focusing on the demos:
+  - The [demos READMEs](demos/) now contain more guidance and explanation to make it easier to find a relevant example [\#1200](https://github.com/stellargraph/stellargraph/pull/1200)
+  - A [demo for loading data from Neo4j](demos/basics/loading-saving-neo4j.ipynb) has been added [\#1184](https://github.com/stellargraph/stellargraph/pull/1184)
+  - The [demo for link prediction using Node2Vec](demos/link-prediction/random-walks/cora-lp-demo.ipynb) has been rewritten to be clearer [\#1190](https://github.com/stellargraph/stellargraph/pull/1190)
+- New algorithms:
+  - `GCNSupervisedGraphClassification`: supervised graph classification model based on Graph Convolutional layers (GCN) [\#929](https://github.com/stellargraph/stellargraph/issues/929).
+- `DeepGraphInfomax` can be used to train almost any model in an unsupervised way, via the `corrupt_index_groups` parameter to `CorruptedGenerator` [\#1243](https://github.com/stellargraph/stellargraph/pull/1243). Additionally, `GraphSAGE` ([\#1162](https://github.com/stellargraph/stellargraph/pull/1162)) and `HinSAGE` ([\#1254](https://github.com/stellargraph/stellargraph/pull/1254)) provide defaults and so can be trained without manually specifying this parameter (in addition to existing supported algorithms like `GCN` and `GAT`).
```

Comment:
would it be valuable to explicitly call out here that DGI HinSAGE now allows unsupervised training on heterogeneous graphs?